src/main/scala/chisel3/ltl/LTL.scala

```diff
@@ -412,7 +364,7 @@ sealed abstract class AssertPropertyLike {
     apply(Sequence.BoolSequence(cond), Some(clock), Some(disable), Some(label))
   }
 
-  def createIntrinsic(label: Option[String]): Instance[VerifAssertLikeIntrinsic]
+  def createIntrinsic(label: Option[String]): (Bool) => Unit
```

Comment:
We should make these protected or package private and backport a deprecation (since we've made a source incompatible change anyway).